cogs/countdown.py

```diff
@@ -9,8 +9,10 @@
 
 EVENTS = [
     PizzaDaoEvent(5, 22,  "Bitcoin Pizza Day"),
-    PizzaDaoEvent(6, 28, "Tau Day")
+    # PizzaDaoEvent(6, 28, "Tau Day") # disabled until event is official
```

Comment:
Could add a disabled flag here instead. Better practice long term. 
```python
EVENTS = [
    PizzaDaoEvent(5, 22,  "Bitcoin Pizza Day"),
    PizzaDaoEvent(6, 28, "Tau Day",  False),
]

....

if event.enabled and event.days_until_event <= days_until_next_event:
```

Reply:
agreed! any chance you have a favorite book or article on best design patterns / practices as they pertain to python?